homeassistant/components/itach/remote.py

```diff
@@ -106,22 +117,22 @@ def is_on(self):
     def turn_on(self, **kwargs):
         """Turn the device on."""
         self._power = True
-        num_repeats = kwargs.get(ATTR_NUM_REPEATS, 1)
-        self.itachip2ir.send(self._name, "ON", num_repeats)
+        self.itachip2ir.send(self._name, "ON", self._ir_count)
         self.schedule_update_ha_state()
 
     def turn_off(self, **kwargs):
         """Turn the device off."""
         self._power = False
-        num_repeats = kwargs.get(ATTR_NUM_REPEATS, 1)
-        self.itachip2ir.send(self._name, "OFF", num_repeats)
+        self.itachip2ir.send(self._name, "OFF", self._ir_count)
         self.schedule_update_ha_state()
 
     def send_command(self, command, **kwargs):
         """Send a command to one device."""
-        num_repeats = kwargs.get(ATTR_NUM_REPEATS, 1)
+        num_repeats = kwargs.get(ATTR_NUM_REPEATS, DEFAULT_NUM_REPEATS)
         for single_command in command:
-            self.itachip2ir.send(self._name, single_command, num_repeats)
+            self.itachip2ir.send(
+                self._name, single_command, self._ir_count * num_repeats
```

Comment:
Say i had a TV that needed 3 ir count for power but i then sent a command of volume up or such. Will this mean that the volume raises once (ie all commands need the 3 ir) or will it raise volume 3x?

Reply:
As long as ‘ir_count’ is a reasonable value, you won’t get repeat commands sent. It should be device specific, so if you need 3 repeats for power on a particular device, you should need the same number of repeats for one volume increment.